### PR TITLE
v0.5.0 - Update to bevy 0.14

### PR DESCRIPTION
**Pull request by simbleau**
### Added

- New `scene_ui` example demonstrating a `VelloScene` attached to a `bevy::ui::Node`

### Changed

- Updated to bevy 0.14
- Updated to vello 0.2
- Updated to velato 0.3
- Updated to vello_svg 0.3

### Fixed

- Removed `Arc` in another `Arc` for `VelloFont`
- Opacity now correctly applies to SVG assets.
- Opacity now applies correctly to the lottie image group, rather than each element and path within it, causing overdraw.
- `VelloScene` components on `bevy::ui::Node` entities now account for Bevy's UI layout systems and render at the expected viewport coordinates

### Removed

- Pancam and/or egui from all examples besides the demo, as external dependencies can bottleneck upgrading to the next bevy version.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -23,7 +23,7 @@ jobs:
       run: cargo clippy -- -D warnings
 
     - name: Build | Rustfmt
-      run: cargo fmt --all -- --check
+      run: cargo fmt -- --check
```
Comment (waywardmonkeys) [inline .github/workflows/ci.yml:26]: Why are these changes here rather than a separate PR?

File: CHANGELOG.md [hunk 2]
```diff
@@ -10,14 +10,30 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 ## Unreleased
 
+## 0.5.0
+
 ### Added
 
-- New `scene-ui` example demonstrating a `VelloScene` attached to a `bevy::ui::Node`
+- New `scene_ui` example demonstrating a `VelloScene` attached to a `bevy::ui::Node`
```
Comment (waywardmonkeys) [inline CHANGELOG.md:17]: Line is missing a period at the end (based on the period being present in most other lines).

**Review by waywardmonkeys — APPROVED** [review]
Looks fine, most of it is just a boilerplate update for changes in Bevy unless I missed something big. A couple of small nits if you feel like it.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (simbleau) [reply .github/workflows/ci.yml:26]: The examples didn't compile yet because of bevy_pancam not being updated yet
